src/zeit/cms/tagging/tag.py

```diff
@@ -136,12 +110,13 @@ def unique_id_to_tag(unique_id):
 
 class AbsoluteURL(zope.traversing.browser.absoluteurl.AbsoluteURL):
 
-    zope.component.adapts(Tag, zeit.cms.browser.interfaces.ICMSLayer)
+    zope.component.adapts(zeit.cms.tagging.interfaces.ITag,
+                          zeit.cms.browser.interfaces.ICMSLayer)
 
     def __str__(self):
         base = zope.traversing.browser.absoluteURL(
             zope.site.hooks.getSite(), self.request)
-        return base + '/++tag++' + self.context.code
+        return base + '/++tag++' + self.context.code.encode('unicode_escape')
```

Comment:
Anmerkung: Wenn man das auf UTF-8 + Quoting umbauen will, betrifft das auch `zeit.retresco.tag.Tag.uniqueId` (neben `unique_id_to_tag` & `AbsoluteURL` & `TagTraverser` in `zeit.cms.tagging.tag`).